frontstage/templates/surveys/surveys-link-expired.html

```diff
@@ -0,0 +1,18 @@
+{% extends "layouts/_block_content.html" %}
+{% from "components/breadcrumb/_macro.njk" import onsBreadcrumb %}
+{% from "components/fieldset/_macro.njk" import onsFieldset %}
+{% from "components/input/_macro.njk" import onsInput %}
+{% from "components/lists/_macro.njk" import onsList %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% set page_title = "survey link has expired" %}
+{% block main %}
+    {% if is_transfer %}
+        <h1 class="u-fs-xl">Transferred survey link has expired</h1>
```

Comment:
Should this be 'Transfer survey link has expired'?  I don't actually know, just asking

Reply:
As per the card thats what it says
<transferred survey link expired>

Page title: Transferred survey link has expired
Page content:
This link has expired.

If you still need access to the surveys, you should ask your colleague to transfer the surveys again.